docs/features/job-level-affinity.md

```diff
@@ -0,0 +1,147 @@
+# Quickstart - Job Level Affinity
+This Quickstart guide provides a step-by-step tutorial on how to effectively use the job-level affinity feature for podgroups, with a focus on both preferred and required affinity types. For comprehensive information about this feature, please consult the [Job Level Affinity Design Document](../functionality/job-level-affinity/job-level-affinity-overview.md).
```

Comment:
link is broken. Please double check.

Reply:
This will be valid after Shuo's MR is checked in. But let me remove this and add a TODO instead.

Reply:
Updated.